### PR TITLE
kill.sh script for kill bombardier process

### PR DESCRIPTION
**Pull request by JekaNS**

### CODE REVIEW

File: kill.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	shift 1
+
+	for ip in ${@}
+	do
+		process ${key} ${ip} &
+	done
+}
+
+process(){
+	echo "${2}"
```
Comment (alex-ander-is) [inline kill.sh:30]: Does it make sense to list what's killed when we are killing everything eveywhere, so do the wipe?

File: kill.sh [hunk 2]
```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+main(){
+	./updateInstancesIPs.sh
+
+	if [ -f STOCKHOLM_INSTANCES_IPS.txt ]
+	then
+		echo "Stockholm kill by:"
```
Comment (alex-ander-is) [inline kill.sh:8]:
```suggestion
		echo "Killing all bombardiers at Stockholm..."
```

File: kill.sh [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+		killbombardier "key-stockholm-1.pem" "$(cat STOCKHOLM_INSTANCES_IPS.txt)"
+	fi
+
+	if [ -f FRANKFURT_INSTANCES_IPS.txt ]
+	then
+		echo "Frankfurt kill by:"
```
Comment (alex-ander-is) [inline kill.sh:14]:
```suggestion
		echo "Killing all bombardiers at Frankfurt..."
```

File: kill.sh (hunk 1 above)
Comment (JekaNS) [reply kill.sh:30]: It's not required, but I'm personally want to see where processes was killed.

**Review by alex-ander-is — APPROVED** [review]

**Pull request merged**